docs/UserGuide.md

```diff
@@ -72,14 +72,22 @@ return back to [Table of Contents](#table-of-contents)
 
 ## Installing MediConnect
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have Java `11` or above installed in your Computer. If you do not have it installed yet, refer to the following guides based on your operating system:
```

Comment:
optional: include how to check if you have java 11 and exactly 11 installed. this might be abit too tedious also

Reply:
I added a link to check java version. Java should be backwards compatible (though not tested).

Reply:
that will be sufficient